Source/aweXpect.Reflection/Filters/ConstructorFilters.WhichAre.cs

```diff
@@ -23,4 +23,52 @@ public static Filtered.Constructors WhichAreNot(this Filtered.Constructors @this
 		=> @this.Which(Filter.Prefix<ConstructorInfo>(
 			type => !type.HasAccessModifier(accessModifier),
 			"non-" + accessModifier.GetString(" ")));
+
+	/// <summary>
+	///     Filters for constructors that are public.
+	/// </summary>
+	public static Filtered.Constructors WhichArePublic(this Filtered.Constructors @this)
```

Comment:
Also the source files should be split up. One separate file for each expectation name and its negations, so e.g. `WhichArePublic` and `WhichAreNotPublic` should be in file `ConstructorFilters.WhichArePublic.cs`